net-misc/r8152/r8152-2.16.3-r1.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit udev linux-info linux-mod
+
+DESCRIPTION="r8152 driver for Realtek USB FE / GBE / 2.5G Gaming Ethernet Family Controller"
+HOMEPAGE="https://www.realtek.com/en/component/zoo/category/network-interface-controllers-10-100-1000m-gigabit-ethernet-usb-3-0-software"
+
+SRC_URI="http://rtitwww.realtek.com/rtdrivers/cn/nic1/${P}.tar.bz2"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 ~x86"
```

Comment:
Don't commit with stable keywords:
* https://devmanual.gentoo.org/keywording/index.html#keywording-on-upgrades
* `pkgcheck scan --commits` should warn on this too

Reply:
Yeah, pretty clear I committed the wrong ebuild file. Must have screwed something up while copying things around during testing. Will re-test and push the correct copy.